macros/edr/data_monitoring/schema_changes/get_columns_changes_query.sql

```diff
@@ -77,7 +77,7 @@
             pre.data_type as pre_data_type,
             pre.detected_at
         from cur inner join pre
-            on (cur.full_table_name = pre.full_table_name and cur.column_name = pre.column_name)
+            on (cur.full_table_name = pre.full_table_name and lower(cur.column_name) = lower(pre.column_name))
```

Comment:
Thanks @sadahry!
We should probably add the same safety to the full_table_name as well. 
Could you add to the PR?

Reply:
Sure! fixed https://github.com/elementary-data/dbt-data-reliability/pull/290/commits/9a405310e1a1d5c1baf1c9005c93ada1b20bbf65 .